nuvla/job_engine/connector/helm_driver.py

```diff
@@ -103,11 +107,19 @@ def _from_absolute_url(self, op: str, release: str,
             values_yaml_fd.close()
         return result
 
-    def _from_helm_repo_cred(self, op, helm_repo, helm_repo_cred, chart_name,
-                             version, values_yaml, helm_release, namespace):
-        repo_name = 'helm-repo'
+    def _from_helm_repo_cred(self, op, helm_repo_url, helm_repo_cred,
+                             chart_name, version, values_yaml, helm_release,
+                             namespace):
+        repos_config_fd = None
+        registry_config_fd = None
+        env = {}
 
-        repos_config = f"""
+        cmd = [op, helm_release, os.path.join(helm_repo_url, chart_name)]
+
+        if helm_repo_url.startswith('http'):
```

Comment:
You could branch into smaller fns, one for http and one for oci

Reply:
Yes, I thought about this as well. OK - I will extract two methods.